temporian/implementation/numpy/data/test/event_set_test.py

```diff
@@ -69,12 +80,76 @@ def test_repr(self):
 """,
         )
 
-    def test_memory_usage(self):
-        memory_usage = self.evset.memory_usage()
-        print("memory_usage:", memory_usage)
+    def test_repr_limits(self):
+        config.max_printed_features = 1
+        config.max_printed_indexes = 1
+        # Numpy summarization only makes sense for limit >= 6
+        config.max_printed_events = 6
+        evset = event_set(
+            timestamps=[0.1, 0.2, 0.3, 0.4, 0.5, 0.6, 0.7, 0.8],
+            features={
+                "a": [1, 2, 3, 7, 8, 9, 10, 11],
+                "b": [4, 5, 6, 9, 10, 15, 16, 17],
+                "x": [1, 1, 1, 1, 1, 1, 1, 2],
+                "y": ["h", "h", "h", "h", "h", "h", "h", "w"],
+            },
+            indexes=["x", "y"],
+        )
+        self.assertEqual(
+            repr(evset),
+            """indexes: [('x', int64), ('y', str_)]
+features: [('a', int64), ('b', int64)]
+events:
+    x=1 y=b'h' (7 events):
+        timestamps: [0.1 0.2 0.3 ... 0.5 0.6 0.7]
+        'a': [ 1  2  3 ...  8  9 10]
+        ...
+    ... (showing 1 of 2 indexes)
+memory usage: 1.2 kB
+""",
+        )
 
-        self.assertLessEqual(memory_usage, 1200 + 500)
-        self.assertGreaterEqual(memory_usage, 1200 - 500)
+    def test_html_repr_no_limits(self):
+        config.max_display_indexes = 0
+        config.max_display_features = 0
+        config.max_display_events = 0
+
+        self.assertEqual(
+            self.evset._repr_html_(),
+            "<div><h3>Index: (x=1, y=hello)</h3>"
+            + "3 events × 2 features"

```

Comment:
Since the number of features is the same for all the indexes, maybe we don't need to display it each time.

Reply:
I agree. Though changing this required adding text common to the whole eventset.
Took the chance and included number of indexes and memory usage.
Final result (setting small max_feats and indexes on purpose, only for this demo):
<img width="1085" alt="Screen Shot 2023-08-07 at 13 13 24" src="https://github.com/google/temporian/assets/12506292/d56d44a4-e207-4428-814e-a5cc3f630796">